common/openapi.yml

```diff
@@ -83,8 +83,8 @@ paths:
         500:
           $ref: '#/components/responses/ServerError'
     post:
-      summary: Create new pipeline job
-      operationId: createJob
+      summary: Create new pipeline run
+      operationId: createPipelineRun
```

Comment:
do we need to update the integration test script to call this endpoint after it gets merged?

Reply:
discussed on slack - not due to this change - the API path hasn't changed - but we DO want to make a new path, a new group of endpoints for /api/pipelinerun/ rather than having this under /api/pipelines/. thanks!

Reply:
this PR description now contains a link to the e2e test change PR